### PR TITLE
Upgrade to Storybook 8

### PR DESCRIPTION
**Pull request by ShaunEvening**
## What changed
- Upgrade to Node 18
- Upgrade to Storybook 8 alpha
- Update build settings to match latest changes in addon-kit
- Drop Babel for SWC in example Storybook

# Version

Published prerelease version: `v2.3.0-next.2`

<details>
  <summary>Changelog</summary>

  #### 🐛 Bug Fix
  
  - Use bundling strategy suitable for Storybook 8 [#107](https://github.com/chromaui/storybook-addon-pseudo-states/pull/107) ([@JReinhold](https://github.com/JReinhold))
  
  #### ⚠️ Pushed to `next`
  
  - Upgrade to SB v8.0.0 (non-rc) and fix reference to Icons component ([@ghengeveld](https://github.com/ghengeveld))
  - Merge branch 'main' into next ([@ghengeveld](https://github.com/ghengeveld))
  - Restore Babel for Jest ([@ghengeveld](https://github.com/ghengeveld))
  - Bump major version ([@ShaunEvening](https://github.com/ShaunEvening))
  - Update build settings ([@ShaunEvening](https://github.com/ShaunEvening))
  - Upgrade to SB 8; Use SWC instead of babel ([@ShaunEvening](https://github.com/ShaunEvening))
  - Use node 18 ([@ShaunEvening](https://github.com/ShaunEvening))
  
  #### Authors: 3
  
  - Gert Hengeveld ([@ghengeveld](https://github.com/ghengeveld))
  - Jeppe Reinhold ([@JReinhold](https://github.com/JReinhold))
  - Shaun Evening ([@ShaunEvening](https://github.com/ShaunEvening))
</details>

<details>
  <summary>📦 Published PR as canary version: <code>3.0.0--canary.100.fce90b1.0</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install storybook-addon-pseudo-states@3.0.0--canary.100.fce90b1.0
  # or 
  yarn add storybook-addon-pseudo-states@3.0.0--canary.100.fce90b1.0
  ```
</details>

### CODE REVIEW

**Review by ghengeveld — APPROVED** [review]

**Comment by MacroMan** [comment]
When can we expect a release please?

**Comment by ghengeveld** [comment]
It should've automatically released a new version, but that seems to have [failed](https://github.com/chromaui/storybook-addon-pseudo-states/actions/runs/8420645901/job/23055776408). I'll take a look at it but that likely won't be today.

**Comment by ghengeveld** [comment]
@MacroMan v3.0.0 is released now 👍

File: src/manager/PseudoStateTool.tsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React, { useCallback } from "react"
-import { Icons, IconButton, WithTooltip, TooltipLinkList } from "@storybook/components"
```
Comment (wegry) [inline src/manager/PseudoStateTool.tsx:2]: Is this the only declaration of `Icons`? Does [this line](https://github.com/chromaui/storybook-addon-pseudo-states/pull/100/files#diff-a0545e75046f4ff07d0014505213b181ffe6cc6b01e916c0545e5f54d7b60581R65) need updated as well?

Comment (ghengeveld) [reply src/manager/PseudoStateTool.tsx:2]: Yeah 👍

**Pull request merged**